### PR TITLE
REVIEW - METS root element requirments

### PR DESCRIPTION
**Pull request by carlwilson**
- improvements for clarity; and
- paticular attention to attributes for content types.

### CODE REVIEW

**Comment by carlwilson** [comment]
Please wait until after the A3 group meeting 09/05 before merging.

**Comment by karinbredenberg** [comment]
CSIP62 and CSIP63 also needs to be updated to be consistent with this change.

File: profile/CSIP.xml [hunk 1]
```diff
@@ -166,8 +166,8 @@
             </requirement>
             <requirement ID="CSIP2" REQLEVEL="MUST" RELATEDMAT="VocabularyType" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
                 <description>
-                    <head>General content type</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">The @type attribute must be used to provide a general description of the content information type of the package. A fixed vocabulary is used which will evolve under the curation of the DILCIS Board as additional content information type specifications are developed.</p>
+                    <head>Content type</head>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The @TYPE attribute must be used to declare the content information type of the package. Legal values are defined in a fixed vocabulary. When the content type used falls outside of the defined vocabulary the @TYPE value must be set to "OTHER". The vocabulary will evolve under the curation of the DILCIS Board as additional content information type specifications are developed.</p>
```
Comment (koit) [inline profile/CSIP.xml]:
```suggestion
                    <p xmlns="http://www.w3.org/1999/xhtml">The @TYPE attribute MUST be used to declare the type of content held in the package, e.g. book, journal, stereograph, dataset, video, etc. (legal values are defined in a fixed vocabulary). When the content type used falls outside of the defined vocabulary, the @TYPE value must be set to "OTHER" and the specific value declared in @csip:OTHERTYPE. The vocabulary will evolve under the curation of the DILCIS Board.</p>
```

- I borrowed the list of examples from mets.xsd, aiming to give the reader some broad understanding of what kind of "type" is being discussed, without the need to look up the actual values from the vocabulary
- a standardised term for payload content would be welcome here: "content information," "content files," "payload data" etc. Consistent use would make it easier, as pretty much any term requires explanation, e.g. "content files" can be seen to cover both data and metadata, as both are the IP's content
- shortened the last sentence as `@TYPE` is not related to content information type specifications

File: profile/CSIP.xml [hunk 2]
```diff
@@ -176,8 +176,8 @@
             </requirement>
             <requirement ID="CSIP3" REQLEVEL="SHOULD" RELATEDMAT="VocabularyType" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
                 <description>
-                    <head>Other general content type</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">The @csip:OTHERTYPE attribute must be used for stating the general type of the package (genre) when @TYPE has the value "OTHER"</p>
+                    <head>Extended content type</head>
```
Comment (koit) [inline profile/CSIP.xml]: Maybe `Custom content type`? "Extended content type" sounds a bit like "Type of extended content," as if we had "normal" content and "extended" content. Although, "custom" suffers from the same problem as "Type of custom content."

But essentially it is custom, as the attribute is for recording a custom type-description value that is not present in the official vocabulary.

File: profile/CSIP.xml [hunk 3]
```diff
@@ -176,8 +176,8 @@
...
+                    <p xmlns="http://www.w3.org/1999/xhtml">When the @TYPE attribute has the value "OTHER" the @csip:OTHERTYPE attribute must be used to the content type of the package or representation.</p>
```
Comment (koit) [inline profile/CSIP.xml]:
```suggestion
                    <p xmlns="http://www.w3.org/1999/xhtml">When the @TYPE attribute has the value "OTHER", the @csip:OTHERTYPE attribute MUST be used to declare the content type of the package/representation.</p>
```

File: profile/CSIP.xml [hunk 4]
```diff
@@ -187,7 +187,7 @@
             <requirement ID="CSIP4" REQLEVEL="SHOULD" RELATEDMAT="VocabularyContentInformationTypeSpecification" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
                <description>
                     <head>Specific content type</head>
```
Comment (koit) [inline profile/CSIP.xml]:
```suggestion
                    <head>Content Information Type Specification</head>
```

File: profile/CSIP.xml [hunk 5]
```diff
@@ -187,7 +187,7 @@
...
-                   <p xmlns="http://www.w3.org/1999/xhtml">An added attribute which describes the specific content information type specification used for the transferred content. The attribute is mandatory to use when the METS document describes a representation. The vocabulary is going to evolve under the care of the DILCIS Board as additional content information type specifications are developed.</p>
+                   <p xmlns="http://www.w3.org/1999/xhtml">Used to declare the specific content information type specification used for the transferred content. The attribute is mandatory for representation level METS documents. The vocabulary will evolve under the care of the DILCIS Board as additional content information type specifications are developed.</p>
```
Comment (koit) [inline profile/CSIP.xml]:
```suggestion
                    <p xmlns="http://www.w3.org/1999/xhtml">Used to declare the Content Information Type Specification that was used in creating the package. Legal values are defined in a fixed vocabulary. The attribute is mandatory for representation level METS documents. The vocabulary will evolve under the care of the DILCIS Board as additional Content Information Type Specifications are developed.</p>
```

>mandatory for representation level METS documents

There are only a few CIT specifications, so many cases will not have a suitable CITS: a private person's documents (i.e. not managed with an ERMS), a photo collection, a film (incl. the script, storyboard, costume sketches, audio, video, trailers, posters etc.). So the attribute is mandatory, but CITS itself is not.

I propose adding the value "NONE" to the vocabulary. The terms refer to specific CITS, so "OTHER" is not a logically correct value for none. "OTHER" would go for an in-house CITS, e.g. if a film archive develops one for internal use with films.

"NONE" would not trigger the use of `@csip:OTHERCONTENTINFORMATIONTYPE`, only "OTHER" would do it.

File: profile/CSIP.xml [hunk 6]
```diff
@@ -187,7 +187,7 @@
             <requirement ID="CSIP4" REQLEVEL="SHOULD" RELATEDMAT="VocabularyContentInformationTypeSpecification" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
```
Comment (koit) [inline profile/CSIP.xml]: I would change the vocabulary name from:

`VocabularyContentInformationTypeSpecification`

to:

`VocabularyContentInformationType`

to match the attribute name.

File: profile/CSIP.xml [hunk 7]
```diff
@@ -1374,7 +1374,7 @@
             <p xmlns="http://www.w3.org/1999/xhtml">Do we need a note?</p>
         </note>
     </tool>
-    <Example ID="metsRootElementExample2" LABEL="METS root element example with content not in the value list @TYPE being transferred">
+    <Example ID="metsRootElementExample1" LABEL="METS root element showing use of csip:@OTHERTYPE attribute when an appropriate package content type value isnt' available in the vocabulary. The @TYPE attribute value is set to OTHER.">
```
Comment (koit) [inline profile/CSIP.xml]:
```suggestion
    <Example ID="metsRootElementExample1" LABEL="METS root element showing use of csip:@OTHERTYPE attribute when an appropriate package content type value is not available in the vocabulary (the @TYPE attribute value is set to OTHER).">
```

File: profile/CSIP.xml [hunk 8]
```diff
@@ -1387,7 +1387,7 @@
             xsi:schemaLocation="http://www.loc.gov/METS/ http://www.loc.gov/standards/mets/mets.xsd http://www.w3.org/1999/xlink http://www.loc.gov/standards/mets/xlink.xsd https://dilcis.eu/XML/METS/CSIPExtensionMETS https://dilcis.eu/XML/METS/CSIPExtensionMETS/DILCISExtensionMETS.xsd">
          </mets:mets>
     </Example>
-    <Example ID="metsRootElementExample3" LABEL="METS root element example with transferred content of a type from the fixed value list and the representation content is not part of the value list, but is instead a value decided upon in the submission agreement.">
+    <Example ID="metsRootElementExample2" LABEL="METS root element illustrating the use of a custom csip:@OTHERCONTENTINFORMATIONTYPE attribute value when an appropriate content type value could not be found in the vocabulary. The csip:@CONTENTINFORMATIONTYPE attribute value is set to OTHER.">
```
Comment (koit) [inline profile/CSIP.xml]:
```suggestion
    <Example ID="metsRootElementExample2" LABEL="METS root element illustrating the use of a custom csip:@OTHERCONTENTINFORMATIONTYPE attribute value when the correct value for the Content Information Type Specification does not exist in the vocabulary (the csip:@CONTENTINFORMATIONTYPE attribute value is set to OTHER).">
```

I made the wording more strict, as CITS is a very specific document that has to be referenced precisely. "Appropriate value could not be found" sounds as if it was OK to find an approximate match.

**Pull request merged**